lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1648,6 +1649,33 @@ def _resource_sku_to_capability(  # noqa: C901
                 ]
             )
 
+        # add vm which support nested virtualization
+        # https://docs.microsoft.com/en-us/azure/virtual-machines/acu
+        if resource_sku.family in [
+            "standardDv3Family",
+            "standardDSv3Family",
+            "standardDv4Family",
+            "standardDSv4Family",
+            "standardDDv4Family",
+            "standardDDSv4Family",
+            "standardEv3Family",
+            "standardESv3Family",
+            "standardEv4Family",
+            "standardESv4Family",
+            "standardEDv4Family",
+            "standardEDSv4Family",
+            "standardFSv2Family",
+            "standardMSFamily",
+            "standardMSMediumMemoryv2Family",
```

Comment:
there are so many vm size support nested-virtualization. e.g.
https://docs.microsoft.com/en-us/azure/virtual-machines/fx-series
https://docs.microsoft.com/en-us/azure/virtual-machines/m-series

Reply:
They all would be covered since they belong to the same family